src/System.Private.DataContractSerialization/src/System.Private.DataContractSerialization.csproj

```diff
@@ -11,7 +11,7 @@
     <DebugSymbols>true</DebugSymbols>
     <PlatformTarget>AnyCPU</PlatformTarget>
     <DefineConstants>$(DefineConstants);FEATURE_SERIALIZATION</DefineConstants>
-    <DefineConstants Condition="'$(TargetGroup)'=='uapaot'">$(DefineConstants);NET_NATIVE</DefineConstants>
```

Comment:
It should stay as uapaot only in the serializer projects. You may just keep it alone for this one. There is separate issue about it: https://github.com/dotnet/corefx/issues/15892

(When I was looking at this earlier I just searched for NETNATIVE - not for NET_NATIVE - so I have missed this one. Sorry...)